### PR TITLE
handle properly older version of ansible

### PR DESCRIPTION
**Pull request by zikalino**

### CODE REVIEW

**Comment by zikalino** [comment]
test failures are unrelated to this change

**Comment by Fred-sun** [comment]
@yungezz

**Comment by yungezz** [comment]
@yuwzho  this failure https://travis-ci.org/Azure/azure_preview_modules/jobs/486723319 is the one I mentioned to you, managed disk test failure. It's now a consistent failure in role CI, blocks general changes. could you pls look at it?

**Comment by yungezz** [comment]
@zikalino  https://travis-ci.org/Azure/azure_preview_modules/jobs/486723321 request  error, could you pls  take a look?

**Comment by zikalino** [comment]
@yungezz now the tests are fixed, could you approve this pr?

**Comment by zikalino** [comment]
@yungezz pls approve, now the failures are fixed in same pr.

**Comment by Fred-sun** [comment]
@yungezz

File: module_utils/azure_rm_common.py [hunk 1]
```diff
@@ -14,7 +14,10 @@
 from os.path import expanduser
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.ansible_release import __version__ as ANSIBLE_VERSION
+try:
+    from ansible.module_utils.ansible_release import __version__ as ANSIBLE_VERSION
+except:
+    ANSIBLE_VERSION='unknown'
```
Comment (yungezz) [inline module_utils/azure_rm_common.py]: why it's 'unknown'

**Pull request merged**